views/layouts/email-2022.hbs

```diff
@@ -68,7 +82,8 @@
     <tr class="footer">
       <td style="border-top: 1px solid #ddd; padding: 24px 0">
         <p>{{{ getString "email-2022-unsubscribe" unsubscribe-link-attr=( concat "href=" this.SERVER_URL ) }}}</p>
-        <p>{{{ getString "email-2022-hibp-attribution" hibp-link-attr='href="https://haveibeenpwned.com/" rel="noopener"' }}}
+        <p>{{{ getString "email-2022-hibp-attribution" hibp-link-attr='href="https://haveibeenpwned.com/"
+          rel="noopener"' }}}
         </p>
         <img src="{{ SERVER_URL }}/img/email_images/mozilla-logo-bw-rgb.png" width="130px"
```

Comment:
Can you add an `alt="Mozilla"` to the logo image `mozilla-logo-bw-rgb.png` as well?